app/views/home/show.html.haml

```diff
@@ -1 +1,128 @@
+          .caption.right-align
+            %h3 Right Aligned Caption
+            %h5.light.grey-text.text-lighten-3 Here's our small slogan.
+      %li
+        %img{:src => "http://lorempixel.com/580/250/nature/4"}
+          / random image
+          .caption.center-align
+            %h3 This is our big Tagline!
+            %h5.light.grey-text.text-lighten-3 Here's our small slogan.
+.row
+  .col.s10.offset-s1
+    .row
+      .col.s4
+        .row.col.s12.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+      .col.s4
+        .row.col.s12.categories
+          = image_tag("shirt.png", alt: "A Fine Shirt")
+      .col.s4
+        .row.col.s12.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+    .row
+      .col.s5
+        %hr
+      .col.s2.categoryHeading BESTSELLERS
+      .col.s5
+        %hr
+    .row
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+    .row
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+    .row
+      .col.s5
+        %hr
+      .col.s2.categoryHeading NEW ARRIVALS
+      .col.s5
+        %hr
+    .row
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+    .row
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+      .col.s3
+        .row.categories
+          = image_tag("tshirt.png", alt: "A Fine Shirt")
+        .row sdkjfdjkd
+.col.s12.promotion
```

Comment:
As discussed, footer should be in the layout, either as a partial or in the page itself.